_script/requirements

```diff
@@ -16,12 +16,21 @@ function upgrade_go_using_gimme() {
 function install_go_dependencies() {
     echo "INFO: Installing go dependencies"
     go get ./...
+    install_coveralls
+}
+
+function install_coveralls() {
+    echo "INFO: Installing coveralls dependencies"
+    go get github.com/mattn/goveralls
 }
 
 function prepare_dir_permissions() {
     echo "INFO: preparing gopath/output folder permissions"
-    sudo chown travis /home/travis/gopath/src/github.com
-    sudo chmod o+w /home/travis/gopath/src/github.com/dachad/tcpgoon/out/
+    USER="travis"
```

Comment:
is there any specific reason for this?

Reply:
Yes. Given these directories are "created" because of the mountpoint with our source code, they are all owned by root, while we want CI to run as travis. This sets the minimum set of permissions in the GOPATH to allow this user to install the packages we need